build/images/ovs/Dockerfile.ubi

```diff
@@ -32,7 +32,12 @@ RUN cd /tmp/openvswitch* && \
     sed -e "s/@VERSION@/$OVS_VERSION/" rhel/openvswitch-fedora.spec.in > /tmp/ovs.spec && \
     yum-builddep -y /tmp/ovs.spec && ./boot.sh && \
     ./configure --prefix=/usr --localstatedir=/var --sysconfdir=/etc && \
-    make rpm-fedora && mkdir -p /tmp/ovs-rpms && \
+    # https://github.com/openvswitch/ovs/blob/v2.17.7/rhel/openvswitch-fedora.spec.in#L26-L27
+    # Antrea will run OVS and logrotate as the root user. Disabling libcapng helps
+    # to skip configuring the user and group for OVS and logrotate.
```

Comment:
Before this change, OVS runs as openvswitch, which user logrotate uses?

Reply:
Without this change, OVS runs as root but logrorate will su to openvswitch when performing log rotation. We did not use the `--ovs-user` option to specify the OVS user.

Reply:
Could you help me understand why the issue doesn't exist in ubuntu image, where I see OVS also runs as root?

Reply:
For OVS deb packages the logrotate configuration does not switch users before rotation by default.
https://github.com/openvswitch/ovs/blob/v2.17.9/debian/openvswitch-switch.logrotate#L2
But the configuration of rpm packages will switch to non-root users if libcap is linked. 
The line https://github.com/openvswitch/ovs/blob/v2.17.9/rhel/etc_logrotate.d_openvswitch#L9 will be changed in https://github.com/openvswitch/ovs/blob/v2.17.9/rhel/openvswitch-fedora.spec.in#L325

Reply:
Thanks for the explanation. I feel we could rephrase it a little:
```
logrotate needs to run as the same user as OVS to get the proper permissions of log files.
As Antrea runs OVS as root, we disable libcapng to make logrotate also run as root. 
See https://github.com/openvswitch/ovs/blob/v2.17.7/rhel/openvswitch-fedora.spec.in#L26-L27.
```

Reply:
updated. Thanks!